README.md

```diff
@@ -20,22 +20,90 @@ Virtink consists of 3 components:
 
 **NOTE**: Virtink is still a work in progress, its API may change without prior notice.
 
-## Prerequisites
+## Installation
+
+Prerequisites:
+
+- Kubernetes 1.16 ~ 1.24
+- [cert-manager](https://cert-manager.io/) 1.0 ~ 1.8. You can install it with `kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.8.2/cert-manager.yaml`
 
-Virtink relies on [cert-manager](https://cert-manager.io/) 1.0 or above for SSL certificate management. You can install cert-manager as follows:
+Install all Virtink components:
 
 ```bash
-kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.8.0/cert-manager.yaml
+kubectl apply -f https://github.com/smartxworks/virtink/releases/download/v0.8.0/virtink.yaml
 ```
 
-## Installation
+Once you have deployed Virtink, you can [create your virtual machines](#create-a-vm).
+
+## Getting Started
+
+### Create a VM
+
+Apply the following manifest to Kubernetes. Note it uses a container disk and as such doesn’t persist data.
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: virt.virtink.smartx.com/v1alpha1
+kind: VirtualMachine
+metadata:
+  name: ubuntu-container-disk
+spec:
+  instance:
+    memory:
+      size: 1Gi
+    disks:
+      - name: ubuntu
+      - name: cloud-init
+    interfaces:
+      - name: pod
+  volumes:
+    - name: ubuntu
+      containerDisk:
+        image: smartxworks/virtink-container-disk-ubuntu
+    - name: cloud-init
+      cloudInit:
+        userData: |-
+          #cloud-config
+          password: password
+          chpasswd: { expire: False }
+          ssh_pwauth: True
+  networks:
+    - name: pod
+      pod: {}
+EOF
+```
+
+Like starting pods, it will take some time to pull the image and start running the VM. You can wait for the VM become running as follows:
 
-Virtink can be installed as follows:
+```bash
+kubectl wait vm ubuntu-container-disk --for jsonpath='{.status.phase}'=Running
+```
+
+### Access the VM (via SSH)
+
+The easiest way to access the VM is via a SSH client inside the cluster. You can access the VM created above as follows:
```

Comment:
an example of accessing VM by a kubernetes Service might be better,  user can use this Service in cluster or outside(NodePort now), and don't need care about vm pod name or IP which is a implementation detail.

Reply:
I'm afraid that would cause an incorrect impression that an extra Service is mandatory to access the VM